src/pages/documentation/getting-started/get-the-sdk.md

```diff
@@ -0,0 +1,108 @@
+import AddDependenciesIosACP from './tabs/get-the-sdk/add-dependencies/ios-acp.md'
+import AddDependenciesIosAEP from './tabs/get-the-sdk/add-dependencies/ios-aep.md'
+import AddDependenciesAndroid from './tabs/get-the-sdk/add-dependencies/android.md'
+import AddInitializationCodeIosACP from './tabs/get-the-sdk/add-initialization-code/ios-acp.md'
+import AddInitializationCodeIosAEP from './tabs/get-the-sdk/add-initialization-code/ios-aep.md'
+import AddInitializationCodeAndroid from './tabs/get-the-sdk/add-initialization-code/android.md'
+import GetTheSdkIosACP from './tabs/get-the-sdk/get-the-sdk/ios-acp.md'
+import GetTheSdkIosAEP from './tabs/get-the-sdk/get-the-sdk/ios-aep.md'
+import GetTheSdkAndroid from './tabs/get-the-sdk/get-the-sdk/android.md'
```

Comment:
@calebk1 Is there any reason for splitting these samples up in separate files? This may become very difficult to maintain or update over time, since we now need to go to each individual file for each small sample code, per platform.
For general docs like "Getting started" I am thinking we can even use in-line code samples if we don't anticipate those will come from an external source and not add additional files under tabs. 

And we should explore using this concept in the context of importing content from external repos (e.g. the open source extension repo) in the future for things specific to an extension (e.g. extension getting started, api reference, event reference). 

Can you also find out if these can be applied:
1. can we include multiple git dependencies as long as they use the markdown files with variant definition ?
2. we can declare multiple variants per section, one for api name and one for language, e.g:
```
<Variant apiname="get-queue-size" platform="ios-aep" />

// usage
<API query="apiname=get-queue-size&platform=ios-aep" />
```

If that does not work, in the meantime can we have one tabs definition file per page and use variants to distinguish between individual tabs and apis.
For example for Analytics ext you can have tabs/api-reference.md and inside of it we have multiple variants for all apis/languages:
```
<Variant apitab="android-get-queue-size" />
// Android code here
<Variant apitab="ios-aep-get-queue-size" />
// AEP code here
```
Usage:
```
import API from './tabs/api-reference.md'
<TabsBlock orientation="horizontal" slots="heading, content" repeat="3"/>

Android

<API query="apitab=android-get-queue-size" />

iOS (AEP 3.x)

<API query="apitab=ios-aep-get-queue-size" />
...
```
More details on the examples proposed: https://github.com/adobe/aio-theme#embedding-markdown-documents-and-filtering-content

Reply:
With regards to the points made here...

## Tabs

With the way tabs work on the Gatsby format, it, unfortunately, needs to be put into separate files on a per-tab basis. 

That being said, I don't think it's _terribly_ difficult to maintain, since the locations of the file structure is very clear, IMO, and it's just putting the location of the tab within its own file, rather than within the file it was created in. The only difficulty comes in the creation of the files, since you'd need to create them all, but maintaining it really isn't very difficult.

## Importing external files

While this _is_ possible, I think that this, if anything, will actually make it a lot harder to maintain, since you'll be introducing the following variables:

1. Different locations for different repositories, which means that knowing which places need to be updated makes it tricky for me to maintain.
2. Files are imported on a per-run basis, which means that every time the file is updated, I have to re-export the documentation and get the new link of what's imported and update the documentation with what the new imports are.
3. Lack of easy control in the documentation - sometimes, if we need to make changes that would make the import fit better on the dev.adobe-side, it would be a lot trickier to manage.

For these reasons, I'd advise against just importing from the external documents, since we'd be, in my opinion, making it harder to maintain as well as making it less flexible for the documentation to look like.

## Using variants with tabs

This might work? I'll have to investigate this, but I won't be able to do this until post-October since I just do not have the time to investigate this in time for the first release.

I hope this answers all your questions?